main.go

```diff
@@ -0,0 +1,279 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/sliceutil"
+	"github.com/bitrise-steplib/bitrise-step-android-unit-test/cache"
+	"github.com/bitrise-tools/go-android/gradle"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	"github.com/bitrise-tools/go-steputils/tools"
+	shellquote "github.com/kballard/go-shellquote"
+)
+
+const (
+	apkEnvKey      = "BITRISE_APK_PATH"
+	testApkEnvKey  = "BITRISE_TEST_APK_PATH"
+	testSuffix     = "AndroidTest"
+	apkPathPattern = "*/build/outputs/apk/*.apk"
```

Comment:
please let the user change this: https://github.com/bitrise-steplib/bitrise-step-android-build/blob/master/step.yml#L49